bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -62,8 +62,14 @@ spec:
             if [[ ! -S /run/openvswitch/db.sock ]]; then
               systemctl_restart ovsdb-server
             fi
+            # We need to explicitly exit on SIGTERM, see https://github.com/openshift/cluster-dns-operator/issues/65
+            function quit {
+                exit 0
+            }
+            trap quit SIGTERM
             # Don't need to worry about restoring flows; this can only change if we've rebooted
-            exec tail -F /host/var/log/openvswitch/ovs-vswitchd.log /host/var/log/openvswitch/ovsdb-server.log
+            tail --pid=$BASHPID -F /host/var/log/openvswitch/ovs-vswitchd.log /host/var/log/openvswitch/ovsdb-server.log &
```

Comment:
Question: why does `exec tail ...` not exit when it gets a SIGTERM? Shouldn't that always exit? Bash is gone...

Reply:
By default in Linux (possibly Unix in general) pid 1 has `SIGTERM` set to ignore by default...  For basically bad reasons this is still the default with pid namespaces; it clearly would have been better to change it but we can't now.

https://vagga.readthedocs.io/en/latest/pid1mode.html
https://hynek.me/articles/docker-signals/
etc.

Reply:
OHHHHHHHH. Wow.

Reply:
(frantically checking my other pods...)